pulsar/io.sls

```diff
@@ -0,0 +1,20 @@
+{% from "pulsar/map.jinja" import pulsar with context %}
+{% set mirror_data = 'https://www.apache.org/dyn/closer.cgi/pulsar?as_json=1'|http_query %}
+{% set mirror_urls = mirror_data.body|load_json %}
+{% set mirror_url = mirror_urls.preferred %}
+{% set mirror_backup = mirror_urls.backup[0] %}
+
```

Comment:
Would it better to have the mirror info in `map.jinja` since it's used here and the install file?

Reply:
Fair point, will move.